src/trio/testing/_fake_net.py

```diff
@@ -466,15 +466,16 @@ async def send(self, data: Buffer, flags: int = 0) -> int:
 
     @overload
     async def sendto(
-        self, __data: Buffer, __address: tuple[object, ...] | str | Buffer
+        self, data: Buffer, address: tuple[object, ...] | str | Buffer, /
     ) -> int: ...
 
     @overload
     async def sendto(
         self,
-        __data: Buffer,
-        __flags: int,
-        __address: tuple[object, ...] | str | None | Buffer,
+        data: Buffer,
+        flags: int,
+        address: tuple[object, ...] | str | None | Buffer,
+        /,
```

Comment:
yeahno this will give typing errors. There's reasons for those double underscores, see comments in the socket file - might be good to copy them here.

Reply:
Fixed in new version with inline entire file ignore with comment